server/src/internalClusterTest/java/org/elasticsearch/cluster/routing/ShardRoutingRoleIT.java

```diff
@@ -537,18 +543,9 @@ public void testSearchRouting() throws Exception {
                     }
                 }
                 BytesReference pitId = client().execute(TransportOpenPointInTimeAction.TYPE, openRequest).actionGet().getPointInTimeId();
-                try {
-                    assertResponse(prepareSearch().setPointInTime(new PointInTimeBuilder(pitId)).setProfile(true), response -> {
-                        var profileResults = response.getProfileResults();
-                        assertThat(profileResults, not(anEmptyMap()));
-                        for (final var profileKey : profileResults.keySet()) {
-                            assertThat(profileKey, in(searchShardProfileKeys));
-                        }
-                    });
-                } finally {
-                    client().execute(TransportClosePointInTimeAction.TYPE, new ClosePointInTimeRequest(pitId));
```

Comment:
Where'd these close calls go?

Reply:
Thanks, @original-brownbear, I forgot about that. I added it around each prepareSearch call, but I'm not sure if this is the right approach. By the way, the test succeeds with that configuration.

Reply:
Thank you @Armin. I expected the test to fail with this update. I'll revert to this class